game/configuration.hh

```diff
@@ -43,6 +43,7 @@ class ConfigItem {
 	std::string const& getShortDesc() const { return m_shortDesc; } ///< get the short description for this ConfigItem
 	std::string const& getLongDesc() const { return m_longDesc; } ///< get the long description for this ConfigItem
 	void addEnum(std::string name); ///< Dynamically adds an enum to all values
+	void removeAllEnums(); /// Removes all the enum values from this entry.
```

Comment:
I'm affraid the naming "enume values" is quite missleading, there is no enum in there, isn't there?

Reply:
So to get back to this. Within our `schema.xml` we define "enums" these "enums" are basicly a label with an integer backing it up. All of the enums defined in there aren't actual Enum class values throughout our C++ codebase.

The name enum for this is poorly choosen or the usage is poorly applied. As for now i think we'll have to live with this. But i can feel a configuration.cc/hh overhaul coming.

Reply:
ok